lib/releaselib.sh

```diff
@@ -50,12 +51,10 @@ release::update_job_cache () {
     logecho -r " $job cache..."
   fi
 
-  run=$($GSUTIL cat $logroot/$job/latest-build.txt 2>/dev/null) || return
-
   for ((cnt=0;cnt<=$cache_limit;cnt++)); do
-    # Once we hit one that is set, break out
+    # Once we hit a JOB[$run] that is set (or run gets to 0), break out
     # the cache should be complete after that
-    [[ -n ${JOB[$run]} ]] && break
+    ((run==0)) || [[ -n ${JOB[$run]} ]] && break
```

Comment:
I think the bug I pointed out last time still exists, FWIW. We set `JOB[$run]` on L61, but then `$run` is decremented on L63, so `JOB[$run]` is always going to be empty. I'm not sure if that's the expected behavior.